sequencer/src/api/sql.rs

```diff
@@ -659,6 +669,12 @@ async fn reward_header_dependencies<Mode: TransactionMode>(
             catchup.add_chain_config(cf);
         };
 
+        let version = header.version();
```

Comment:
I think the above chain config catchup is not needed anymore because we already do it for fee header deps?